gammapy/region/circle.py

```diff
@@ -132,6 +132,21 @@ def __init__(self, pos, radius):
         self.pos = SkyCoord(pos)
         self.radius = Angle(radius)
 
+    @property
+    def area(self):
+        """Circle Area
+        """
```

Comment:
This formula isn't obvious.

Please leave a comment that it's the spherical cap area and a link to:
https://en.wikipedia.org/wiki/Solid_angle#Cone.2C_spherical_cap.2C_hemisphere
https://en.wikipedia.org/wiki/Versine#Definitions
Maybe even putting this in the docstring is better than as a code comment?

This should be equivalent and slightly simpler / faster, no?
```python
val = 2 * np.pi * (1 - np.cos(self.radius))
```